.github/scripts/run-linux-poweroff.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+from fabric.api import *
+
+from common import manager_fsim_dir, manager_hostname
+from ci_variables import ci_workflow_id
+
+def run_linux_poweroff():
+    """ Runs Linux poweroff workloads """
```

Comment:
[On the networked target].  Maybe this can take a config.ini as an argument so we can do the non-networked targets too? (Which frankly are more important.)

Reply:
This is running the `linux-poweroff-all-no-nic.ini` and the `linux-poweroff-nic.ini`. Did you want to run a different set of tests?

Reply:
See L29/30